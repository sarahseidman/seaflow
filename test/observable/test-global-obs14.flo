int apple(int x) {
    printi(x);
    return x;
}

int add(int x, int y) {
    return x + y;
}


int $a = 10;
int $b = 5;

int $c = $a / $b;

subscribe(apple, $c);

$a = 50;
$a = 500;
$b = 100;