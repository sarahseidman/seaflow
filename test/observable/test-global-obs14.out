2
10
100
5
